bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/searchresults/SearchResultsImpl.java

```diff
@@ -11,14 +11,12 @@
  *    governing permissions and limitations under the License.
```

Comment:
This was refactored to use the `SearchResultsService`, which hopefully hides most of the details about how to actually query for search results.